app/src/main/java/io/bluetrace/opentrace/onboarding/SetupCompleteFragment.kt

```diff
@@ -46,7 +46,7 @@ class SetupCompleteFragment : OnboardingFragmentInterface() {
         return inflater.inflate(R.layout.fragment_setup_complete, container, false)
     }
 
-    override fun getProgressValue(): Int = 100
+    override fun getProgressValue(): Int = 75
```

Comment:
what is 100 now?

Reply:
Nothing. The last step will be at 75% and when finished will go directly to home screen.